taroscript/send_test.go

```diff
@@ -469,11 +469,7 @@ func checkPreparedSplitSpend(t *testing.T, spend *taroscript.SpendDelta,
 		spend.InputAssets[prevInput].Amount-addr.Amount,
 	)
 	if spend.InputAssets[prevInput].Amount == addr.Amount {
-		require.Equal(t, spend.NewAsset.Amount, uint64(0))
-		require.Equal(
-			t, asset.ToSerialized(spend.NewAsset.ScriptKey.PubKey),
-			asset.NScriptKey,
-		)
+		require.True(t, spend.NewAsset.IsUnspendable())
```

Comment:
Let's fold this in as a fixup commit (during rebase, squashed with the main commit)for the final version fo this PR?